src/lib/types.ts

```diff
@@ -24,6 +24,7 @@ export interface Node extends DrupalNode {
   opening_hours: any
   picture_url: string
   picture_url_override: any
+  drupal_internal__id: number
 }
```

Comment:
In relation to other comment to this PR about using ts-ignore `drupal_internal__id` and `locationId` should be used as string instead of number.